### DISCUSSION TITLE
Can we please allow http callback urls for ips starting with `192.168`?

### DISCUSSION THREAD

**Discussion by zfei** [fief-dev/fief#144] [Q&A] [answered]
I'm doing continuous integration on a local server. Often times need to try out the frontend from another computer.
I guess whitlisting `192.168.*.*` should be safe?

**Comment by frankie567** [comment, marked as answer]
Well, I guess so 🤔  I'll make some research to be sure we're not introducing any vulnerability doing this.

**Reply by zfei** [reply]
I just went ahead and made pull request on this: #151.
Let me know if this doesn't feel right.

Thanks.
